app/main.py

```diff
@@ -1,3 +1,17 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    stored_results = dict()
+
+    def wrapper(*args) -> Callable:
+
```

Comment:
Mind the code style, you don't have an empty line after assigning the previous function. Remove the empty line here